@here/harp-mapview/lib/text/Placement.ts

```diff
@@ -625,10 +641,6 @@ function placePointLabelAtAnchor(
     tmpBox.copy(label.bounds!);
     tmpBox.min.multiplyScalar(scale);
     tmpBox.max.multiplyScalar(scale);
-    // Add margin after scaling, this ensures the margin is consistent across all
-    // labels - regardless of distance scaling (or any other) factor.
-    // TODO: Make the margin configurable
-    tmpBox.expandByVector(pointLabelMargin);
     tmpBox.translate(textOffset);
```

Comment:
This part was a main problem, margin was applied to bounding box, but text was not moved by its (margin) amount from icon.